.docker/images/nginx/tests/bootstrap.php

```diff
@@ -6,29 +6,48 @@
 
 include_once __DIR__ . '/vendor/autoload.php';
 
-function get_curl_headers($path = "/", $opts = NULL)
-{
-  $uri = getenv('LOCALDEV_URL') ?: 'http://nginx:8080';
-
-  $response = null;
+/**
+ * Execute CURL and get response headers.
+ *
+ * @param string $path
+ *   (optional) URL path to get the headers for. Defaults to '/'.
+ * @param null|string $opts
+ *   (optional) CURL-compatible string of headers. Defaults to NULL.
+ *
+ * @return array
+ *   Array of returned headers.
+ *
+ * @throws \RuntimeException
+ *   If CURL exited with an error.
+ */
+function get_curl_headers($path = '/', $opts = NULL) {
+  $uri = 'http://nginx:8080';
+  $uri = $uri . '/' . ltrim($path, '/');
 
-  $path = '/' . ltrim($path, '/');
+  $response = NULL;
+  exec("docker-compose exec -T test curl -s {$uri} -I {$opts} 2>&1", $response, $ret);
 
-  exec("docker-compose exec -T test curl {$uri}{$path} -I {$opts} 2>/dev/null", $response);
+  if (is_debug()) {
```

Comment:
this allows to debug CURL commands easier if `--debug` is provided as an argument to `phpunit`.